### ISSUE TITLE
Issues Building on Node 10.5

### ISSUE THREAD

**Issue by drebelox** [bengfarrell/nuimotion#1]
```
 gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
  CXX(target) Release/obj.target/nuimotion/Main.o
../Main.cpp:255:5: error: no matching function for call to 'uv_queue_work'
    uv_queue_work(uv_default_loop(), req, frameWorker, onFrameWorkerThreadComplete);
    ^~~~~~~~~~~~~
/Users/drebelox/.node-gyp/0.10.5/deps/uv/include/uv.h:1411:15: note: candidate function not viable: no known conversion
      from 'void (uv_work_t *)' to 'uv_after_work_cb' (aka 'void (*)(uv_work_t *, int)') for 4th argument
UV_EXTERN int uv_queue_work(uv_loop_t* loop, uv_work_t* req,
              ^
1 error generated.
make: *** [Release/obj.target/nuimotion/Main.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/node-gyp/lib/build.js:267:23)
gyp ERR! stack     at ChildProcess.EventEmitter.emit (events.js:98:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:784:12)
gyp ERR! System Darwin 12.3.0
gyp ERR! command "node" "/usr/local/bin/node-gyp" "rebuild"
gyp ERR! cwd /Users/drebelox/Desktop/nuimotion-master/src
gyp ERR! node -v v0.10.5
gyp ERR! node-gyp -v v0.9.5
gyp ERR! not ok
```

**Comment by bengfarrell** [comment]
Thanks much! I haven't upgraded to 0.10 yet (apparently this is a 0.9 issue as well). I did some light research and it looks like this call is simply deprecated in favor of another. The only tricky part will be making sure whatever I do is backwards compatible. I'll try to address this in the next couple of days

**Comment by bengfarrell** [comment]
Again, thanks for pointing that out. I just upgraded to 0.10 tonight, and everything appears shipshape after making a small change to the C++. I went back and published under 0.8 as well, and no problems arose - so the fix looks perfectly backwards compatible. Folks with the same issue had the same code change - and it was just a 0.8 to 0.10 compatibility issue.
